server/form-pages/apply/schema.json

```diff
@@ -1298,6 +1298,55 @@
               "type": "string"
             }
           }
+        },
+        "contingency-plan-partners": {
+          "allOf": [
+            {
+              "type": "object",
+              "properties": {
+                "partnerAgencyName": {
+                  "type": "string"
+                },
+                "namedContact": {
+                  "type": "string"
+                },
+                "phoneNumber": {
+                  "type": "string"
+                },
+                "roleInPlan": {
+                  "type": "string"
+                }
+              }
+            },
+            {
+              "type": "object",
+              "properties": {
+                "saveAndContinue": {
+                  "type": "string"
+                },
```

Comment:
This feels like it shouldn't be part of what gets sent to the API. Though I guess no sensible way around?

Reply:
Yeah there isn't that I can think of as all of the page's 'bodyProperties' are passed to the server.